src/components/IssueController.jsx

```diff
@@ -61,7 +61,7 @@ export default class IssueController extends FalcorController {
           //console.log("Category: " + JSON.stringify(category));
           return(
             <div key={category} className="issue__category">
-              <h2 className="section-header">{category}</h2>
+              <h2 className="section-header">{category.replace('-', ' ')}</h2>
```

Comment:
I don't think it's much extra work, it'll be within the same query. Personally I think it would be more than okay having a category slug field, where we get data is from Falcor, so that's where we should keep on getting it.

If we really want to optimize it and make all slugs just be a replace spaces with hyphens, we can also make Falcor do that itself, and not fetch anything extra. Just write it into the Falcor path that if you fetch name and slug at the same time, return slug as a transform of name.

As of right now I did also add a slug field to the category table in the database though, so if we keep that, we should of course also use it. And that gives more flexibility in choosing slug names than just the transform. We can of course just decide it's always a transform, that's a design decision I guess.